theorems/T000394.md

```diff
@@ -0,0 +1,16 @@
+---
+uid: T000394
+if:
+  and:
+    - P000052: true
+    - P000162: true
+then:
+  P000164: true
+refs:
+- mr: MR1039321
+  name: General Topology (Engelking, 1989)
+- doi: 10.1007/978-1-4615-7819-2
+  name: Rings of Continuous Functions (Gillman and Jerison)
+---
+
+See theorem 12.2 in {{10.1007/978-1-4615-7819-2}} or exercise 3.11.D of {{MR1039321}}.
```

Comment:
Typos: missing "doi:" and "mr:" in the curly braces.
Also possibly missing newline in the last line?  (not entirely sure, but when I edit something and add the final newline, there is usually an extra line (l. 17 here) that shows up as a result)